### PR TITLE
Karl/mobile nav header

### PR DESCRIPTION
**Pull request by Karl-J-Bruder**
Added the mobile nav header (back arrow + current page title)

### CODE REVIEW

File: src/components/atoms/AnimersLogo/styles.tsx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import styled from "styled-components"
+import { colors, fsize, fweight } from "../../../styles/vars"
+
+
+export const StyleLogoContainer = styled.div``;
```
Comment (Alfrex92) [inline src/components/atoms/AnimersLogo/styles.tsx]: If you're not adding styles, there is no need to create `StyleLogoContainer`

File: src/components/atoms/MenuLink/styles.tsx [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import styled from "styled-components"
+import { media } from "../../../styles/breakpoints"
+import { colors, space, fsize } from "../../../styles/vars"
+
+export const StyleNavMenuLink = styled.div`
+    align-items: center;
+    display: flex;
+    min-height: 100px;
```
Comment (Alfrex92) [inline src/components/atoms/MenuLink/styles.tsx:10]: do we need this `min-height`? If this is a button I can see why we need it.

File: src/components/atoms/SNSIcon/index.tsx [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+}
+
+
+
+const SocialIcon: React.FC<SNSIconInterface> = (
+    props: SNSIconInterface
+) => {
+    return (
+        <StyleSocialIconCircle>
+            <StyleSocialIconContent href="#" target="_blank">
```
Comment (Alfrex92) [inline src/components/atoms/SNSIcon/index.tsx]: `href` should come from props. And also, you need to add ` rel="noopener noreferrer" `.

RULE: When you link to a page on another site using the `target="_blank"`  you need to always use ` rel="noopener noreferrer" `

https://web.dev/external-anchors-use-rel-noopener/

File: src/components/molecules/BasicButton/styles.tsx [hunk 4]
```diff
@@ -0,0 +1,24 @@
+import styled from "styled-components"
+import { colors, fsize, fweight } from "../../../styles/vars"
+
+export const StyleBasicButton = styled.button`
+    background-color: ${colors.red.normal};
+    border-radius: 50px;
+    color: #fff;
+    font-size: ${fsize.big};
+    font-weight: ${fweight.w7};
+    height: 40px;
```
Comment (Alfrex92) [inline src/components/molecules/BasicButton/styles.tsx]: do we need the height?

File: src/components/molecules/NavMenu/index.tsx [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+
+import MenuLink from "../../atoms/MenuLink/index"
+
+const Links = translationsEn.menu;
+
+const NavMenu: React.FC = () => {
+    return (
+        <StyleMenuLinkContainer>
+            {Links.map(link => (
+                <MenuLink linkName={link.title} />
```
Comment (Alfrex92) [inline src/components/molecules/NavMenu/index.tsx]: you forgot the KEY hahahaha

You probably need to add a memo in front of your computer or idk. You always forgot the key and you'll have a lot of bugs if you don't add the key.

File: src/components/molecules/NavMenu/styles.tsx [hunk 6]
```diff
@@ -0,0 +1,17 @@
+import styled from "styled-components"
+import { space } from "../../../styles/vars"
+
+export const StyleMenuLinkContainer = styled.div`
+    height: 50vh;
+    margin: ${space.l};
+    min-height: 375px;
+
+    @media (min-width: 768px) {
```
Comment (Alfrex92) [inline src/components/molecules/NavMenu/styles.tsx]: why are you doing this?

This should be something like:

```
 ${media.med`
  
  `}
```
(Don't forget to import media)

File: src/components/organisms/DesktopNav/index.tsx [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+}
+
+const DesktopNav: React.FC<DesktopNavInterface> = (
+    props: DesktopNavInterface
+) => {
+    return (
+        <StyleDesktopNavWrapper>
+            <AnimersLogo />
+            <NavMenu />
+            <div style={{ display: "flex", alignItems: "center" }}>
```
Comment (Alfrex92) [inline src/components/organisms/DesktopNav/index.tsx:23]: inlining styles hahaha

File: src/components/organisms/Footer/index.tsx [hunk 8]
```diff
@@ -0,0 +1,31 @@
...
+    copyrightContent: string
+}
+
+const Footer: React.FC<FooterInterface> = (
+    props: FooterInterface
+) => {
+    return (
+        <StyleFooterWrapper>
+            <CTAEmail message={props.emailMessage} address={props.emailAddress} />
+            <div style={{ width: "150px" }}>
```
Comment (Alfrex92) [inline src/components/organisms/Footer/index.tsx:22]: inlining styles hahaha

File: src/components/pages/MobileNavMenu/index.tsx [hunk 9]
```diff
@@ -0,0 +1,29 @@
...
+    socialMessage: string,
+    copyrightContent: string
+}
+
+const MobileNavMenu: React.FC<MobileNavMenuInterface> = (
+    props: MobileNavMenuInterface
+) => {
+    return (
+        <StyleMobileNavMenuWrapper>
+            <div style={{ height: "10vh", maxWidth: "500px", width: "100vw" }}>
```
Comment (Alfrex92) [inline src/components/pages/MobileNavMenu/index.tsx]: inline styles 😢

File: src/components/atoms/MenuLink/styles.tsx (hunk 2 above)
Comment (Karl-J-Bruder) [reply src/components/atoms/MenuLink/styles.tsx:10]: Looks like it's ok without min-height. Not sure why I used it, actually.

File: src/components/atoms/SNSIcon/index.tsx (hunk 3 above)
Comment (Karl-J-Bruder) [reply src/components/atoms/SNSIcon/index.tsx]: Is this correct?
`<StyleSocialIconContent href={props.url} target="_blank" rel="noopener noreferrer">`

File: src/components/molecules/BasicButton/styles.tsx (hunk 4 above)
Comment (Karl-J-Bruder) [reply src/components/molecules/BasicButton/styles.tsx]: When I remove the height property, the button shrinks. Using min-height also works, so is that better?

File: src/components/molecules/NavMenu/index.tsx (hunk 5 above)
Comment (Karl-J-Bruder) [reply src/components/molecules/NavMenu/index.tsx]: AAHHHH! 🤣 I'm gonna put a sticky note on my monitor right now! lol

File: src/components/molecules/NavMenu/styles.tsx (hunk 6 above)
Comment (Karl-J-Bruder) [reply src/components/molecules/NavMenu/styles.tsx]: Weird. My version doesn't look like that. It has media.med like you wrote.

File: src/components/organisms/Footer/index.tsx (hunk 8 above)
Comment (Karl-J-Bruder) [reply src/components/organisms/Footer/index.tsx:22]: Fixed!

File: src/components/pages/MobileNavMenu/index.tsx (hunk 9 above)
Comment (Karl-J-Bruder) [reply src/components/pages/MobileNavMenu/index.tsx]: Fixed!

**Pull request closed without merging**